core/spec/models/spree/variant_spec.rb

```diff
@@ -270,6 +270,58 @@
 
   end
 
+  describe 'exchange_name' do
+    let!(:variant) { create(:variant, option_values: []) }
+    let!(:master) { create(:master_variant) }
+
+    before do
+      variant.option_values << create(:option_value, {
+                                                     name: 'Foo',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.